crates/bevy_gltf/src/loader.rs

```diff
@@ -1250,12 +1356,30 @@ fn load_node(
                     let primitive_label = primitive_label(&mesh, &primitive);
                     let bounds = primitive.bounding_box();
 
-                    let mut mesh_entity = parent.spawn(PbrBundle {
-                        // TODO: handle missing label handle errors here?
-                        mesh: load_context.get_label_handle(&primitive_label),
-                        material: load_context.get_label_handle(&material_label),
-                        ..Default::default()
-                    });
+                    let mut mesh_entity = if primitive
+                        .extensions()
+                        .map(|extensions| extensions.contains_key("BEVY_meshlet_mesh"))
+                        .unwrap_or(false)
+                    {
+                        if cfg!(feature = "meshlet") {
+                            parent.spawn(MaterialMeshletMeshBundle::<StandardMaterial> {
+                                // TODO: handle missing label handle errors here?
+                                meshlet_mesh: load_context.get_label_handle(&primitive_label),
+                                material: load_context.get_label_handle(&material_label),
+                                ..Default::default()
+                            })
+                        } else {
+                            panic!("glTF file contained a MeshletMesh, but the meshlet cargo feature is not enabled.")
+                        }
```

Comment:
```suggestion
                        #[cfg(feature = "meshlet")]
                        {
                            parent.spawn(MaterialMeshletMeshBundle::<StandardMaterial> {
                                // TODO: handle missing label handle errors here?
                                meshlet_mesh: load_context.get_label_handle(&primitive_label),
                                material: load_context.get_label_handle(&material_label),
                                ..Default::default()
                            })
                        }
                        #[cfg(not(feature = "meshlet"))]
                        {
                            panic!("glTF file contained a MeshletMesh, but the meshlet cargo feature is not enabled.")
                        }
```
not sure this will make it compile, but I think it does

Reply:
That worked, thanks!